src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadAssessmentResource.java

```diff
@@ -115,6 +115,20 @@ public FileUploadAssessmentResource(AuthorizationCheckService authCheckService,
         return super.cancelAssessment(submissionId);
     }
 
+    /**
+     * Delete an assessment of a given submission.
+     *
+     * @param submissionId - the id of the submission for which the current assessment should be deleted
+     * @param resultId     - the id of the result which should get deleted
+     * @return 200 Ok response if canceling was successful, 403 Forbidden if current user is not the assessor of the submission
```

Comment:
For my understanding: It seems that we only throw a badRequest error if the submission does not belong to the result, but do we actually check if the user is the assessor? And If so, shouldn't an admin be allowed to delete it as well?

Reply:
Yeah, I made a mistake in the javadoc there, I changed it.